SU2_CFD/src/CMarkerProfileReaderFVM.cpp

```diff
@@ -45,6 +47,8 @@ CMarkerProfileReaderFVM::CMarkerProfileReaderFVM(CGeometry      *val_geometry,
   filename     = val_filename;
   markerType   = val_kind_marker;
   numberOfVars = val_number_vars;
+  columnNames  = val_columnNames;
+  columnValues = val_columnValues;
```

Comment:
I guess this could be done with an initializer list for the ctor .. but I am unsure if one is to be preferred over the other. The core guidelines dont seem to have a strong opinion either. [Payground](https://gcc.godbolt.org/z/eraGd1qEz)